bin/update_bitmasks

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env python
+
+"""
+Update doc/bitmasks.rst based upon code descriptions
+"""
+
+bitmask_template = """
+=================
+Bit Masks in DESI
+=================
+
+.. NOTE: this file is generated by desidatamodel/bin/update_bitmasks, e.g.
+..   cd desidatamodel/doc
+..   python ../bin/update_bitmasks > bitmasks.rst && make html
+
+This page describes the bitmasks found in DESI files.
+For details on working with these values, please see the tutorial_ on
+that topic.
+
+Redshift Fitting (Redrock) Masks
+--------------------------------
+
+The **ZWARN** bitmask in redshift catalogs indicates known problems with a
+particular redshift fit or associated QA.
+ZWARN==0 is good; any non-zero value indicates a potential problem.
+This mask will be described in more detail in the Redrock paper
+(Bailey et al 2023 in prep), as well as Section 5.3.1 of the
+Survey Operations Paper (Schlafly et al 2023 TODO: add link).
+
+The canonical code location defining these bits is
+`desitarget targetmask.yaml <https://github.com/desihub/desitarget/blob/main/py/desitarget/data/targetmask.yaml#L230>`_.
+Bits 0-15 are set by Redrock itself (the redshift fitter),
+while bits 16-19 are set by DESI-specific post-processing.
+
+**ZWARN Mask Locations**
```

Comment:
I'm not sure why this is simply emphasized text and not a section header.

Reply:
OK, but that means a reference to that subsection does *not* get automatically created. My personal preference is that if you can create a reference to a subsection, you should.